$NetBSD$
pkgsrc doesn't use lib64 or target dirs so don't set them.
fixes build on linux.
http://sf.net/p/qore/code/6817/
--- configure.orig	2014-10-31 21:56:41.605684184 +0000
+++ configure
@@ -15876,14 +15876,6 @@ if test "$enable_64bit" = "yes"; then
       *linux*)	if test "$GXX" = "yes"; then
 		   CXXFLAGS="$CXXFLAGS -m64"
 		fi
-		# only set LIBSUFFIX=64 if /usr/lib64 exists
-		if test -d /usr/lib64; then
-		   LIBSUFFIX=64
-                fi
-		target="${host_cpu}-${host_os}"
-                if test -d /usr/lib/$target; then
-                   LIBSUFFIX=/$target
-                fi
 
 
 		;;
